apache2/vars/main.yaml

```
@@ -4,7 +4,7 @@
 # Pre-processing and validation of parameters.
 
 _a2_conffile_mode: |-
-  {{ "%o" % ((apache2__conffile_mode or 0o644) | int) }}
+  {{ '%o' % ((apache2__conffile_mode or 0o644) | int(base=8)) }}
```

How about going with string once more?:

```suggestion
  {{ '%o' % ((apache2__conffile_mode or '0644') | int(base=8)) }}
```

In Python, `int(x, base=8)` raises if _x_ is numeric; although the same is not true for the Jinja2 filter, I'd suggest helping the reader all the same. 🙃